lib/vquic/ngtcp2.c

```diff
@@ -1660,6 +1660,12 @@ static ssize_t ngh3_stream_send(struct Curl_easy *data,
     return -1;
   }
 
+  // Reset post upload buffer after resumed.
```

Comment:
This style of comment is not allowed in curl code as its not C89 compliant! Use /* */ ...